### ISSUE TITLE
Feeds that episodes are in as-soon-as-media-ready

### ISSUE THREAD

**Issue by cavis** [PRX/feeder.prx.org#1118]
Make a flavor of feed that episodes exist in as-soon-as-possible.  When the audio has been processed.

**Comment by kookster** [comment]
yeah, I think the case here is to have a way to label a feed so when we need to get audio for it earlier then before, it will stitch that audio up (i.e. for Apple DD and Megaphone integrations).
